samples/app/src/main/java/com/example/app/AppDrawer.kt

```diff
@@ -96,6 +97,16 @@ fun AppDrawer(
             },
             modifier = Modifier.padding(NavigationDrawerItemDefaults.ItemPadding)
         )
+        NavigationDrawerItem(
```

Comment:
Instead of adding a new navigation drawer, can we enhance com.example.app.env.EnvViewModel to support discover.